src/lib/modules/console/index.ts

```diff
@@ -112,9 +114,24 @@ class Console {
       this.history.push(cmd);
       this.saveHistory();
     }
-    const pluginCmds = this.plugins.getPluginsProperty("console", "console");
-    for (const pluginCmd of pluginCmds) {
-      const pluginResult = pluginCmd.call(this, cmd, {});
+    const plugins = this.plugins.getPluginsProperty("console", "console");
+    const helpDescriptions = [];
+    for (const plugin of plugins) {
+      // New API
```

Comment:
Shall we deprecate the old API?

Reply:
I think the "new API" comment here can be removed as it doesn't add a lot of value.